src/pudl/metadata/classes.py

```diff
@@ -2004,15 +1975,10 @@ def from_data_source_ids(
             xbrl_resources=xbrl_resources,
         )
 
-    def to_yaml(self, exclude_intermediate_resources: bool = False) -> None:
+    def to_yaml(self) -> None:
         """Output database, table, and column metadata to YAML file."""
         template = _get_jinja_environment().get_template("datasette-metadata.yml.jinja")
-        if exclude_intermediate_resources:
-            [
-                resource
-                for resource in self.resources
-                if not resource.name.startswith("_")
-            ]
```

Comment:
Removing `_out` tables from the metadata.yaml file is not the way to go! Using the [`hidden` argument in the `metadata.yaml` file](https://docs.datasette.io/en/stable/metadata.html#hiding-tables) is the way.